locales/de/common.json

```diff
@@ -57,77 +57,77 @@
-  "Failed to fix NXM handling in Chrome": "Konnte Behandlung von NXM links in Chrome nicht reparieren",
+  "Failed to fix NXM handling in Chrome": "Konnte Verwendung von NXM-Links in Chrome nicht reparieren",
   "Report": "Melden",
   "Success": "Erfolg",
   "Fix was applied": "Erfolgreich repariert",
   "Categories": "Kategorien",
   "Plugins": "Plugins",
   "Save Games": "Spielstände",
   "Show downloads from other games": "Downloads für andere Spiele anzeigen",
-  "Savegame id": "Spielstand id",
+  "Savegame id": "Spielstand-ID",
   "Creation Time": "Zeitpunkt d. Speicherns",
   "Delete": "Löschen",
   "Browse": "Suchen",
   "Meta server": "Metaserver",
-  "Servers to query for meta data.": "Server auf denen nach Metadaten gesucht wird.",
+  "Servers to query for meta data.": "Server, auf denen nach Metadaten gesucht wird.",
   "Add a metaserver": "Metaserver hinzufügen",
   "{{count}} selected": "{{count}} ausgewählt",
-  "Do you really want to delete this mod?": "Wollen Sie diese Mod löschen?",
-  "Do you really want to delete this mod?_plural": "Wollen Sie diese {{ count }} Mods löschen?",
+  "Do you really want to delete this mod?": "Wollen Sie diese Mod wirklich löschen?",
+  "Do you really want to delete this mod?_plural": "Wollen Sie diese {{ count }} Mods wirklich löschen?",
   "Remove Mod": "Mod löschen",
   "Remove Archive": "Archiv löschen",
   "Disable Dependent": "Abhängige Mods deaktivieren",
-  "Confirm deletion": "Löschung bestätigen",
-  "Confirm": "Fortfahren",
+  "Confirm deletion": "Löschen bestätigen",
+  "Confirm": "Bestätigen",
   "Third-party libraries": "Bibliotheken von Drittanbietern",
-  "License": "Lizens",
+  "License": "Lizenz",
   "Released under": "Veröffentlicht unter",
   "External Changes": "Externe Änderungen",
-  "One or more files have been modified on disk since Vortex last activated. You have to decide what happens with them.": "Dateien wurden geändert seit Vortex zuletzt aktiviert hat. Sie müssen entscheiden was damit passiert.",
+  "One or more files have been modified on disk since Vortex last activated. You have to decide what happens with them.": "Eine oder mehrere Dateien wurden seit dem letzten Start von Vortex verändert. Sie müssen bestimmen, was mit ihnen passiert.",
   "These files were modified": "Diese Dateien wurden verändert",
-  "These files were replaced (or removed from the mod)": "Diese Dateien wurden ersetzt (oder in der Mod gelöscht)",
+  "These files were replaced (or removed from the mod)": "Diese Dateien wurden ersetzt (oder aus der Mod entfernt)",
   "These files were deleted": "Diese Dateien wurden gelöscht",
   "Cancel activation": "Aktivierung abbrechen",
   "Confirm changes": "Änderungen bestätigen",
   "Restore": "Wiederherstellen",
   "Category": "Kategorie",
-  "Endorsed": "Endorsed",
+  "Endorsed": "Zu empfehlen",
   "Highlight": "Markierung",
   "Undecided": "Unentschieden",
-  "Abstained": "Abgelehnt",
+  "Abstained": "Nicht zu empfehlen",
```

Comment:
"Abstained" bedeuted nur dass der Nutzer der Mod keine Empfehlung geben will.
"Nicht zu empfehlen" ist "härter", es impliziert dass der Nutzer die Mod schlecht findet.
Wenn der Guide Michelin sich entscheidet einem Restaurant keinen Stern zu geben heißt das noch nicht dass sie von einem Besuch abraten.

Reply:
Gibt es denn einen Unterschied zwischen "Abstained","Undecided" und "N/A", oder haben alle drei dieselbe Wirkung?

Reply:
Jein, alle drei haben natürlich den "Effekt" dass der Benutzer nicht in der Liste der Endorsements auf nexusmods auftaucht.

Aber N/A bedeutet: Vortex weiß überhaupt nicht welche Mod das ist auf nexusmods, das könnte auch eine sein die von einer Fremdseite runtergeladen wurde. Oder der Nutzer hat die Mod endorsed aber weil Vortex die Verbindung nicht herstellen kann weiß es nichts davon.
Bei N/A bietet Vortex den button auch gar nicht an.

Und Abstained ggü. Undecided hat den zusätzlichen Effekt dass diese Mod nicht mehr in dem Dialog auftaucht der bei nexusmods manchmal kommt, von wegen "möchtest du nicht mal die folgenden Mods endorsen".